### ISSUE TITLE
opensource template for api docs 

### ISSUE THREAD

**Issue by gbinal** [18F/mirage#381]
At a future point, I imagine that you'll want to wrap the swagger docs in more api documentation.  I'd encourage re-using one of the [open source docs already in use](https://bit.ly/govapidocs), but if you don't still somehow using open source documentation so that others can re-purpose it.

**Comment by kaitlin** [comment]
The swagger docs are already open source. I'm confused?

**Comment by gbinal** [comment]
A lot of the API UX feedback revolved around further documenting the API (using prose, if you will).  In other words, the difference between [the entire documentation experience of this](http://cfpb.github.io/api/hmda/) versus [just the interactive documentation on this page](http://cfpb.github.io/api/hmda/console/).

**Comment by kaitlin** [comment]
Yes, I think we will be extending and updating the swagger template to make it a more friendly layout with integrated prose. This will be open source since it will be part of the project, which is already open.

**Comment by HobokenMartha** [comment]
Teeny tiny detail, but "concept" is misspelled on this page here:

http://cfpb.github.io/api/hmda/basics.html

/data/{dataset-name}/{contept-name.extension}

**Comment by gbinal** [comment]
> "concept" is misspelled on this page

And the great thing about github-hosted documentation....

https://github.com/cfpb/api/pull/42
